SPECS/libtasn1/libtasn1.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        ASN.1 library
 Name:           libtasn1
-Version:        4.14
```

Comment:
This is a big jump for Mariner 1.0.  Does the patch easily apply?

Reply:
Backported upstream patch instead